openmindat/mindat_api.py

```diff
@@ -245,12 +247,16 @@ def get_mindat_json(self, PARAM_DICT, END_POINT):
                 
                 next_url = response.json()["next"]
                 
-                if next_url:                
-                    try:
-                        response = self.get_results(next_url, json_data, pbar)
-                    except JSONDecodeError:
-                        time.sleep(10)
-                        response = self.get_results(next_url, json_data, pbar)
+                if next_url:
```

Comment:
1. Rename i to attempt for better readability.
2. Remove the unnecessary `pass` statement inside the except block.

Reply:
3. Also print the message indicating that we encountered an error and were retrying, providing the attempting times, e.g., (2/3).
4. Explanation needed: Can you clarify to me if the issue only happens occasionally? According to your error message, can the issue normally be fixed after users really try it again at once? We can discuss this on Slack.